### PR TITLE
SGF Mapping Enhancements

### PR DESCRIPTION
**Pull request by ExistentialEnso**
Allows embedding of maps into the main SGF with filter support, enables setting of custom colors, beta tags, etc.

### CODE REVIEW

File: CmsWeb/Areas/Public/Controllers/SmallGroupFinderController.cs [hunk 1]
```diff
@@ -33,9 +33,11 @@ public ActionResult Index(string id)
 
 				var encoded = Request.Form.ToString();
 
-				foreach (string item in encoded.Split('&'))
+                var loadAllValues = DbUtil.Db.Setting("SGF-LoadAllExtraValues", "false").ToLower() == "true";
```
Comment (drmohundro) [inline CmsWeb/Areas/Public/Controllers/SmallGroupFinderController.cs]: We can use `DbUtil.Db.Setting("SGF-LoadAllExtraValues", false)` instead here, right? (your new overload?)

File: CmsWeb/Areas/Public/Controllers/SmallGroupFinderController.cs [hunk 2]
```diff
@@ -63,8 +65,11 @@ public ActionResult Index(string id)
 			if (sgfm.hasShell())
 				// Process shell here
 				return Content(sgfm.createFromShell());
+			else if (DbUtil.Db.Setting("SGF-UseEmbeddedMap", "false").ToLower() == "true")
```
Comment (drmohundro) [inline CmsWeb/Areas/Public/Controllers/SmallGroupFinderController.cs]: Use your new overload here?

File: CmsWeb/Areas/Public/Models/SmallGroupFinderModel.cs [hunk 3]
```diff
@@ -330,11 +330,15 @@ public void populateFromOrg(Organization org)
 				}
 			}
 
+		    var loadAllValues = DbUtil.Db.Setting("SGF-LoadAllExtraValues", "false").ToLower() == "true";
```
Comment (drmohundro) [inline CmsWeb/Areas/Public/Models/SmallGroupFinderModel.cs]: New strongly-typed overload?

**Comment by ExistentialEnso** [comment]
@drmohundro - The bool settings are now using the overload method.

**Pull request merged**